lib/senders/legacy_index.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (log, config, translator, sender) {
   return require('./templates')()
     .then(function (templates) {
       return {
-        email: new Mailer(translator, templates, config.mail, sender),
+        email: new Mailer(translator, templates, config.smtp, sender),
```

Comment:
Is there any reason for this module to exist, now that the configs are merged? Can we ditch this and switch everything over to just require `lib/senders`?